LoopKit/MessageType.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MessageType.swift
+//  LoopKit
+//
+//  Created by Nathaniel Hamming on 2020-06-23.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+
+public enum MessageType: Int, CaseIterable, Codable {
```

Comment:
`MessageType` is not a great name for this. Perhaps `DeviceErrorState`?

Reply:
Or if this is CGM Specific, then `CGMErrorState`?

Reply:
It isn't always an error. For example, it could be that some action needs to be taken (e.g. calibration). Also, I'm trying to make it independent of device.

What do you think about `DeviceMessageState`?

Reply:
I've refactored the `DeviceManagerStatusReport` to include a `DeviceMessage?` which contains `var message: String` and `var messageState: DeviceMessageState`. Let me know if this is an improvement.